Python/Product/Analysis/Interpreter/Ast/ImportScanner.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
```

Comment:
missing header

Reply:
Huh, this whole file is unused and not meant to be checked in yet... whoops